core/lib/config/src/configs/base_token_adjuster.rs

```diff
@@ -41,7 +78,40 @@ impl BaseTokenAdjusterConfig {
         DEFAULT_CACHE_UPDATE_INTERVAL
     }
 
+    fn default_priority_fee_per_gas() -> u64 {
+        DEFAULT_PRIORITY_FEE_PER_GAS
+    }
+
     pub fn price_cache_update_interval(&self) -> Duration {
         Duration::from_millis(self.price_cache_update_interval_ms)
     }
+
+    pub fn persister_l1_receipt_checking_max_attempts(&self) -> u32 {
+        self.persister_l1_receipt_checking_max_attempts
+            .map_or(DEFAULT_PERSISTER_L1_RECEIPT_CHECKING_MAX_ATTEMPTS, |x| x)
+    }
+
+    pub fn persister_l1_receipt_checking_sleep_duration(&self) -> Duration {
+        Duration::from_millis(
+            self.persister_l1_receipt_checking_sleep_ms
+                .map_or(DEFAULT_PERSISTER_L1_RECEIPT_CHECKING_SLEEP_MS, |x| x),
+        )
+    }
+
+    fn default_max_tx_gas() -> u64 {
+        DEFAULT_MAX_TX_GAS
+    }
+
+    #[deprecated]
+    pub fn private_key(&self) -> anyhow::Result<Option<K256PrivateKey>> {
+        std::env::var("TOKEN_MULTIPLIER_SETTER_PRIVATE_KEY")
```

Comment:
the new function and deprecated? it's a bit of a weird

Reply:
Removed - good spot. Copy / paste from somewhere.